### PR TITLE
Add project status update blogpost

### PR DESCRIPTION
**Pull request by blaberry**
now with funny titles or something

### CODE REVIEW

**Review by kassibuss — DISMISSED** [review]

**Review by brysondev — DISMISSED** [review]

File: src/blog/2024-10-14/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+title: Neotokyo;Rebuild Development Progress Update
+summary: Update on project progress over the last couple of months
+author: kinoko and blaberry 
+
+# Introduction
+2024-10-14 kinoko and blaberry
+
+Hi, it has been quite a few months since the last dev progress update. Better late than never though, so let’s go over what has happened and where the project is heading!
+
+# What’s the state of NT;RE?
```
Comment (nullsystem) [inline src/blog/2024-10-14/index.md]: Can you put `##`/2nd-level headers for all headers and add `#`/1st-level header on top as the title.

**Review by nullsystem — CHANGES_REQUESTED** [review]

File: src/blog/2024-10-14/index.md (hunk 1 above)
Comment (blaberry) [reply src/blog/2024-10-14/index.md]: For sure, ty!

File: src/blog/2024-10-14/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+The state of rebuild is looking great. Most of the parity with the current game has been achieved, with some things still needing to be ironed out, and now the devs have started focusing some attention on the post-parity/QoL stuff. To name a few examples - we’re integrating plugin and competitive mode as native server features, clients have custom crosshair support, there's a new menu UI and we’re looking at implementing native gamemodes previously available as sourcemods. Since there has been a lot added over the months, head over to our website for full patch notes: [https://neotokyorebuild.github.io/](https://neotokyorebuild.github.io/)
```
Comment (nullsystem) [inline src/blog/2024-10-14/index.md]: The reader is already in the website at this point, so no need for the last section or just mention the blog.

File: src/blog/2024-10-14/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+# How can I try the game?
+
+You can try the game any time! We have a guide for installing the game - [https://neotokyorebuild.github.io/guide/install/](https://neotokyorebuild.github.io/guide/install/)
```
Comment (nullsystem) [inline src/blog/2024-10-14/index.md]: Can simplify to: `We have a [guide for installing the game](/guide/install).`

**Review by nullsystem — CHANGES_REQUESTED** [review]

File: src/blog/2024-10-14/index.md (hunk 2 above)
Comment (blaberry) [reply src/blog/2024-10-14/index.md]: True true, makes sense

File: src/blog/2024-10-14/index.md (hunk 3 above)
Comment (blaberry) [reply src/blog/2024-10-14/index.md]: mmm! also true.

File: src/blog/2024-10-14/index.md (hunk 1 above)
Comment (blaberry) [reply src/blog/2024-10-14/index.md]: Done

File: src/blog/2024-10-14/index.md (hunk 2 above)
Comment (blaberry) [reply src/blog/2024-10-14/index.md]: done

File: src/blog/2024-10-14/index.md (hunk 3 above)
Comment (blaberry) [reply src/blog/2024-10-14/index.md]: Done

**Review by blaberry — COMMENTED** [review]
Also added more info on feature requests.

**Review by nullsystem — APPROVED** [review]

**Pull request merged**